### DISCUSSION TITLE
Is there a sponsor / donation link anywhere?

### DISCUSSION THREAD

**Discussion by IsaiahByDayah** [guasam/electron-react-app#50] [General]
Seriously, this is what Electron Forge ["React + TypeScript"](https://www.electronforge.io/guides/framework-integration/react-with-typescript) _SHOULD_ have been. This repo is awesome and has helped me fix the default forge template and hit the ground running. 

Anyways, I wanted to know if there's a "support" link somewhere that I just can't find? Something like a GitHub Support, Kofi, Patreon, etc? Would love to throw a couple of bucks to show my appreciation + help keep this project going 🙏🏾 😅

**Comment by codesbiome** [comment]
Hello @IsaiahByDayah, Its really nice to know this project been helpful! 

Would like to ask a few questions if you don't mind: 
- Which version of this project being used "main" or "minimal"?
- Which OS (windows/mac) are you using, if on Mac, did you notice any errors or issues when running this project? #47 

<br>

Regarding the support link, there is a Sponsor button (Kofi page) added on repo if its needed.

Thank you for using this project & supporting it.
Have a good weekend!

**Reply by IsaiahByDayah** [reply]
Ahh, I see the link on desktop now. Was on the GH mobile app before so maybe that was the issue

and sure, happy to answer those questions:

> Which version of this project being used "main" or "minimal"?

Originally I created a new project with electron forge (webpack+typescript template). After trying to get that working how I’d like I stumbled upon this repo. Because I already had _some_ things setup (tailwind, `electron-tRPC`, a few components, some types & interfaces, etc), I ended up cloning the `minimal` branch into a new directory and copied over the core items piece by piece from there (`tools`, the tsconfig, the eslint config, etc) updating things to fit my file names and other configs (postcss + tailwind for example)

So while I didn’t use it directly as a template you would probably say I used the `minimal` branch for my project. I have looked at `main` though and will likely reference how you do things there as I continue to build out my project

> Which OS (windows/mac) are you using, if on Mac, did you notice any errors or issues when running this project?

For what it’s worth, I’m on an Apple Silicon (M1) mac. Because I’m not (yet?) using any of the window/menu stuff in the `main` branch, I’m not sure if that’s valuable info though ha

The only “issue” I ran into is that it seems the [eslint aliases](https://github.com/codesbiome/electron-react-webpack-typescript-2023/blob/minimal/.eslintrc.json) are missing entries on the `minimal` branch. Not necessarily “breaking” but alias importing wasn’t fully working until I added in the missing ones there

anyways, thanks again for building and sharing this. It’s been incredibly helpful and a valuable resource into getting my feet wet with electron 🙏🏾

**Comment by codesbiome** [comment]
> For what it’s worth, I’m on an Apple Silicon (M1) mac. Because I’m not (yet?) using any of the window/menu stuff in the main branch, I’m not sure if that’s valuable info though ha

That's totally fine, just whenever you use the `main` branch of this project, if you can provide a feedback regarding the layout of titlebar & menus etc (just to know how they appear in Mac OS), that'd be really useful for us all.

<br>

> The only “issue” I ran into is that it seems the [eslint aliases](https://github.com/codesbiome/electron-react-webpack-typescript-2023/blob/minimal/.eslintrc.json) are missing entries on the minimal branch. Not necessarily “breaking” but alias importing wasn’t fully working until I added in the missing ones there

Its nice ya pointed it out, patched it in latest version of minimal branch.

<br>

> anyways, thanks again for building and sharing this. It’s been incredibly helpful and a valuable resource into getting my feet wet with electron 🙏🏾

You're welcome, very glad to know the project is such a helpful resource for you.
Thank you for the support, its really appreciated! ☕👍🏼
